app/controllers/orders_controller.rb

```diff
@@ -1,7 +1,11 @@
 class OrdersController < ApplicationController
 
   def index
-    @orders = Order.all
+    @orders = Order.all.order(date: :desc)
+  end
+
+  def show
+    @order = Order.find(params[:id])
```

Comment:
this is duplicated in 4 methods. 

A common practice for rails controllers is to create a private `set_order` method and add a 
```ruby
before_action :set_order, only: [:show, :edit, :update, :destroy]
```
 callback